### PR TITLE
RobloxFanboy86#8963 + demoman boober#2239 (copy of PR #2565 but now its from upstream)

### PR DESCRIPTION
**Pull request by hammmurabi**
-Full content for Mandate of Palestine, including full Mandate collapse content for Palestine and Israel.

Reposting now that i'm a contributor and have access like you told me to.
Requesting review - DM me on discord for issues

### CODE REVIEW

File: common/on_actions/r56_on_actions.txt
Comment (marijn211) [inline common/on_actions/r56_on_actions.txt:1]: At some point changes on the main to this file have been discarded, needs to be undone

File: common/decisions/PAL.txt
Comment (marijn211) [inline common/decisions/PAL.txt:1]: I removed copy pasted formable decisions with minimal edits that caused error log, instead, use if-clauses in the availability checks of the original decision to make them different specificly for Palestine if that's all your changing

**Comment by TheBrit7** [comment]
Some graphical file names such as portraits and focus icons are inconsistent and do not follow our standard naming process.

File: common/on_actions/r56_on_actions.txt
Comment (hammmurabi) [reply common/on_actions/r56_on_actions.txt:1]: resolved

**Review by marijn211 — APPROVED** [review]

**Pull request merged**